svg/elements/feImage.json

```
@@ -61,10 +61,12 @@
                 "version_added": "12"
               },
               "firefox": {
-                "version_added": true
+                "version_added": true,
+                "notes": "Document fragments (including references to fragments in current document) are not supported (see <a href='https://bugzil.la/455986'>bug 455986</a>)."
```

nit: I think it reads better with a definite article here. Sorry, this is a very small one

```suggestion
                "notes": "Document fragments (including references to fragments in the current document) are not supported (see <a href='https://bugzil.la/455986'>bug 455986</a>)."
```